conda/recipes/cuml/meta.yaml

```diff
@@ -35,6 +35,7 @@ requirements:
     - cudatoolkit {{ cuda_version }}.*
     - ucx-py {{ minor_version }}
     - ucx-proc=*=gpu
+    - hdbscan
```

Comment:
Totally missed this file change in the approval. This is not required to build, only to test, no? If so, then this needs to be removed (and we wouldn’t need ops approval then)